package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "license": "OSL-3.0",
   "engines": {
-    "node": ">=12 <13"
+    "node": "12"
```

Comment:
"node": ">=12 <13"

Reply:
it's the same thing, `>=12 <13` equals to `12`, it's easier to read